packages/paste-website/src/pages/utilities/grid/index.mdx

```diff
@@ -0,0 +1,533 @@
+            <Paragraph>Card content</Paragraph>
+          </Card>
+        </Column>
+      </Grid>
+    </Box>
+  </Do>
+  <Dont title="Don't" body="Don’t use Grid with only one column of content. This causes unnecessary elements. Box is what you’re looking for here.">
+    <Box padding="space60">
+      <Grid gutter="space30">
+        <Column>
+          <Paragraph>Unnecessary HTML elements. Use <SiteLink to="/primitives/box">Box</SiteLink> instead.</Paragraph>
+        </Column>
+      </Grid>
+    </Box>
+  </Dont>
+</DoDont>
+
+<DoDont>
+  <Do title="Do" body="Use Grid to lay out UI elements into consistent columns.">
+    <Box padding="space60">
+      <Grid gutter="space30">
+        <Column>
+          <Card>
+            <Paragraph>Card content</Paragraph>
+          </Card>
+        </Column>
+        <Column>
+          <Card>
+            <Paragraph>Card content</Paragraph>
+          </Card>
+        </Column>
+      </Grid>
+      <Box marginTop="space50">
+        <Grid gutter="space30">
+          <Column>
+            <Card>
+              <Paragraph>Card content</Paragraph>
+            </Card>
+          </Column>
+          <Column>
+            <Card>
+              <Paragraph>Card content</Paragraph>
+            </Card>
+          </Column>
+        </Grid>
+      </Box>
+    </Box>
+  </Do>
+  <Dont title="Don't" body="Don’t let columns grow too small. Gutters should never be wider than columns.">
+    <Box padding="space60">
+      <Grid gutter="space120">
+        <Column>
+          <Card>
+            <Paragraph>Card content</Paragraph>
+          </Card>
+        </Column>
+        <Column>
+          <Card>
+            <Paragraph>Card content</Paragraph>
+          </Card>
+        </Column>
+        <Column>
+          <Card>
+            <Paragraph>Card content</Paragraph>
+          </Card>
+        </Column>
+      </Grid>
+    </Box>
+  </Dont>
+</DoDont>
+
+---
+
+## Usage Guide
+
+### API
+
+#### Installation
+
+```bash
+yarn add @twilio-paste/grid
+```
+
+#### Usage
+
+```jsx
+import {Grid, Column} from '@twilio-paste/grid';
+
+<Grid>
+  <Column>
+    Foo
+  </Column>
+</Grid>
+```
+
+#### Props
+
+| Prop      | Type                                         | Description                                                                                                                         | Default |
+| ----------| ---------------------------------------------| ----------------------------------------------------------------------------------------------------------------------------------- | ------- |
+| gutter?   | `ResponsiveValue<keyof ThemeShape['space']>` | Sets the spacing on both sides of a Column. Use a spacing token that represents half of the total space you need between 2 Columns. |         |
+| vertical? | `boolean, ResponsiveValue<boolean>`          | Stacks the Columns                                                                                                                  |         |
```

Comment:
I never really was a fan of this prop name, reading it now I kind of wonder if it should be named `stack` or `stackColumns`.  Scope creep, I know :P